pkg/ml/ml.go

```diff
@@ -784,25 +784,12 @@ func NewTensor(ctx *Context, dt DType, dims uint32, ne0, ne1, ne2, ne3 uint32, d
 	return &result
 }
 
-// ggml_permute
```

Comment:
I'd prefer to store original GGML function names for easier navigation between projects :)